client/sites/components/sites-dataviews/actions.tsx

```diff
@@ -67,7 +67,16 @@ export function useActions(): Action< SiteExcerptData >[] {
 
 			{
 				id: 'settings',
-				label: __( 'Site settings' ),
+				label: () => {
```

Comment:
Label should always be a string, it's used for accessible labels... and not just for rendering. It's not the right place to add a "record event". It can be used in other places like filtering...

So I would revert this personally.

Reply:
@okmttdhr Maybe just treat this as acceptable loss for now, if someone really needs it we can figure something else out later.

Reply:
Yeah, this is fragile. Actions are not tied to DataViews, they can be used anywhere. That the label is rendered doesn't mean that the DataViews actions menu was opened, it could be that the action was rendered elsewhere.

Reply:
I’ve removed the event for this PR for now. We’ll need to determine the best way to add a callback when the action is opened in a later iteration.